src/plugins/intel_cpu/src/nodes/fullyconnected.cpp

```diff
@@ -61,6 +63,16 @@ bool FullyConnected::isSupportedOperation(const std::shared_ptr<const ov::Node>&
 FullyConnected::FullyConnected(const std::shared_ptr<ov::Node>& op, const GraphContext::CPtr context)
     : Node(op, context, FCShapeInferFactory(op)),
       errorPrefix("FullyConnected node with name '" + getName() + "'") {
+    if (context->getCPUStreamExecutor()) {
```

Comment:
Could you please isolate all the tensor parallel related logic in scope of a separated function

Reply:
> Could you please isolate all the tensor parallel related logic in scope of a separated functions

done